provider_shopper/README.md

```diff
@@ -1,27 +1,35 @@
 ![Realm](https://github.com/realm/realm-dart/raw/main/logo.png)
 
-# provider_shopper
+# Provider Shopper
 
-## A modified Flutter sample application that uses Realm instead of in-memory collection.
+A modified Flutter sample application that uses Realm instead of in-memory collection.
+
+This is a Flutter sample app from [Flutter sample repo](https://github.com/flutter/samples/tree/master/provider_shopper) that shows a state management approach. This app is modified to be integrated with a local realm using [Realm Flutter](https://www.mongodb.com/docs/realm/sdk/flutter/).
```

Comment:
I am not sure this is correct. We are not showing state management approach in this sample. Rather we are replacing the in-memory collection with Realm persistence.

Reply:
I suggest we remove this whole section here